integrationTests/p2p/networkSharding/networkSharding_test.go

```diff
@@ -0,0 +1,153 @@
+package networkSharding
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/stretchr/testify/assert"
+)
+
+var p2pBootstrapStepDelay = 2 * time.Second
+
+func TestConnectionsInNetworkSharding(t *testing.T) {
+	//TODO remove this skip
+	t.Skip("this test fails for the moment")
```

Comment:
cold be better to delay the merge until the test is able to pass, otherwise you'll just need an additional PR just to remove the tow lines